ci-operator/config/mtc-qe/mtc-e2e-qev2/mtc-qe-mtc-e2e-qev2-master__mtc1.8-ocp4.14-lp-rosa-hypershift.yaml

```diff
@@ -0,0 +1,88 @@
+base_images:
+  cli:
+    name: "4.14"
+    namespace: ocp
+    tag: cli
+  mtc-apps-deployer:
+    name: mtc-apps-deployer
+    namespace: mtc-qe
+    tag: master
+  mtc-python-client:
+    name: mtc-python-client
+    namespace: mtc-qe
+    tag: master
+  openshift-cli-installer:
+    name: openshift-cli-installer
+    namespace: ci
+    tag: latest
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+images:
+- context_dir: .
+  dockerfile_path: dockerfiles/interop/Dockerfile
+  inputs:
+    cli:
+      paths:
+      - destination_dir: .
+        source_path: /usr/bin/oc
+    mtc-apps-deployer:
+      paths:
+      - destination_dir: .
+        source_path: /mtc-apps-deployer.tar.gz
+    mtc-python-client:
+      paths:
+      - destination_dir: .
+        source_path: /mtc-python-client.tar.gz
+  to: mtc-runner
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.14"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: mtc-interop-aws-rosa
+  cron: 0 6 25 10 *
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      CLUSTER1_CONFIG: name-prefix=mtc-source;platform=hypershift;region=us-east-2;version=4.13;channel-group=nightly;compute-machine-type=m5.xlarge;timeout=2h;replicas=2
```

Comment:
based on last rehearsal - 'Cluster version 4.13 not found for stream nightly'
should the config be updated?

Reply:
@Simranpal we have some discussion regarding this issue in [this thread](https://redhat-internal.slack.com/archives/C04PK4QPSR1/p1708975693481159), if you're interested